packages/block-editor/src/components/block-list/content.scss

```diff
@@ -134,14 +133,6 @@ _::-webkit-full-page-media, _:future, :root .has-multi-selection .block-editor-b
 	}
 }
 
-.is-block-moving-mode.block-editor-block-list__block-selection-button {
```

Comment:
The BlockSelectionButton has been removed, we're just using the regular toolbar.